libs/cde-visualization/src/lib/components/metadata/metadata.component.html

```diff
@@ -20,7 +20,16 @@
           [cdkConnectedOverlayOpen]="tooltipOpen"
           [cdkConnectedOverlayOffsetX]="5"
         >
-          <div>Visualization metadata for the sample dataset. Sample files may be viewed in Google Sheets.</div>
+          <hra-tooltip-card
+            small
+            [content]="[
```

Comment:
Move content into a readonly variable on the component class and use that instead of embedding complex objects in the html